src/helpers/single-value-providers.ts

```
@@ -0,0 +1,56 @@
+    createCustomFieldIntegerValueProvider: (customField: string): CustomFieldIntegerValueProvider => {
+        return {
+            type: 'custom field integer',
+            customField: customField,
+            getValue: async () => {
+                if (!customField) return undefined;
+
+                const participantInfo = await MyDataHelps.getParticipantInfo();
+                if (participantInfo.customFields.hasOwnProperty(customField)) {
+                    const valueAsString = participantInfo.customFields[customField];
+                    const valueAsInt = parseInt(valueAsString);
+                    if (!isNaN(valueAsInt)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)